server/config.js

```diff
@@ -0,0 +1,7 @@
+let config = {};
```

Comment:
should use const here - generally favour const over let, especially if you don't want to rebind the value 

Also, may be cleaner to write it like this:
```
const config = {
    mongoURI: process.env.MONGO_URI || "mongodb://localhost", 
    mongoPort: process.env.MONGO_PORT || "27017",
    mongoDB = process.env.MONGO_DB || "mail-sender-dev"
}
```

Reply:
Ofcourse! I should have thought before copying it verbatim from stackoverflow :)
Fixed.